### ISSUE TITLE
Add generation of QC report to scpca nextflow pipeline

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/alsf-scpca#108]
After the counts matrices are obtained for each sample, we will want to generate a QC report. We will want to add a process to the nextflow pipeline that takes the counts matrices (or `SingleCellExperiment` objects) as input and outputs a QC report.

**Comment by jashapiro** [comment]
I'm going to suggest that the main functions for generating this report should live in https://github.com/AlexsLemonade/scpcaTools Then we can build that package into the scpca-r docker image and generate the QC reports with a pretty minimal workflow.

**Comment by allyhawkins** [comment]
Yes agreed. There is an issue in scpcaTools for generating the function(s) that will actually be used to generate the QC report as well. Including that here for reference: https://github.com/AlexsLemonade/scpcaTools/issues/7

**Comment by jashapiro** [comment]
Noting that this will happen in the new https://github.com/AlexsLemonade/scpca-nf repo.
